manifests/k/Kitware/ParaView/5.10.0-Windows-Python3.9-msvc2017-AMD64/Kitware.ParaView.yaml

```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Kitware.ParaView
+PackageVersion: 5.10
```

Comment:
```suggestion
PackageVersion: 5.10.0-Windows-Python3.9-msvc2017-AMD64
```

Reply:
I changed this to your suggestion (thanks btw!)